test/OrchardCore.Tests/Shell/ShellContainerFactoryTests.cs

```diff
@@ -131,7 +132,13 @@ public void WhenHostSingletonAndScoped_GetServices_Returns_CorrectImplementation
             Assert.IsType<HostScopedOfTheSameTypeAsSingleton>(services.ElementAt(1));
         }
 
-        private ShellBlueprint CreateBlueprint()
+        private static readonly ShellSettings _uninitializedDefaultShell = new()
```

Comment:
Can we move it to fields area at the top

Reply:
Okay will do